docbrown/core/src/eval.rs

```diff
@@ -10,7 +10,7 @@ use std::{
 use rustc_hash::FxHashSet;
 
 use crate::{
-    tgraph::{TemporalGraph, VertexView},
+    tgraph::{TemporalGraph, VertexReference},
```

Comment:
Should we call it `VRef` or `VertexRef` ?

Reply:
`VertexRef` and `EdgeRef` it is for now